Theoneste1/My-Brand-Theoneste
# Ft about page p lra0v c1

#### What does this PR do?
 create about me page
#### Description of Task to be completed
create index.html
create styles.css
javaScript for some task
#### How should this be manually tested?
Clone this git repo:https://github.com/Theoneste1/My-Brand-Theoneste.git
open command prompt.
Enter in a folder called "My-Brand-Theoneste": cd My-Brand-Theoneste
Enter in a folder called "UI": cd UI
Enter in folder called "about-page" : cd about-page
Look for a file called "index.html, and open it in your favourite browser.
Then you will be able to see, the output
#### Any background context you want to provide?
The stories about which includes the skills, works and experiences
#### What are the relevant stories?
https://trello.com/c/pLRA0vC1/1-about-us-page
#### Screenshots (if appropriate)
![about us image](https://user-images.githubusercontent.com/43376343/88172874-f125e100-cc21-11ea-897e-d860e5da27ba.PNG)
